recipes/taocpp-pegtl/3.x.x/conanfile.py

```diff
@@ -51,28 +50,27 @@ def lazy_lt_semver(v1, v2):
 
         minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
         if not minimum_version:
-            self.output.warn("{} {} requires C++17. Your compiler is unknown. Assuming it supports C++17.".format(self.name, self.version))
+            self.output.warn(f"{self.ref} requires C++17. Your compiler is unknown. Assuming it supports C++17.")
```

Comment:
self.output.warn doesn't exist in conan v2, it's self.output.warning, but I advice to drop this warning, it's useless.